src/sagemaker/image_uris.py

```diff
@@ -14,14 +14,25 @@
 from __future__ import absolute_import
 
 import json
+import logging
 import os
 
 from sagemaker import utils
 
+logger = logging.getLogger(__name__)
```

Comment:
There are difference in `logger` definitions in [various](https://github.com/aws/sagemaker-python-sdk/blob/zwei/src/sagemaker/tensorflow/estimator.py#L30) [parts](https://github.com/aws/sagemaker-python-sdk/blob/zwei/src/sagemaker/model.py#L25) of [the](https://github.com/aws/sagemaker-python-sdk/blob/zwei/src/sagemaker/model_monitor/model_monitoring.py#L90) [code](https://github.com/aws/sagemaker-python-sdk/blob/zwei/src/sagemaker/utils.py#L41). What is your vision on this?

Reply:
I don't think I had a vision for this - just went with whatever looked reasonable after doing a `grep -r "logger =" src` 😅